src/Modules/Shared/Microsoft.PowerShell.Host/Microsoft.PowerShell.Host.psd1

```diff
@@ -10,5 +10,5 @@ FunctionsToExport = @()
 CmdletsToExport="Start-Transcript", "Stop-Transcript"
```

Comment:
@adityapatwardhan, your last commit had 30 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Validate Update-Help from the Web for one PowerShell module..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for one PowerShell module for user scope..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate 'Save-Help -DestinationPath for all PowerShell modules..Validate Save-Help for the 'Microsoft.PowerShell.Core' module
```powershell
Expected 'Microsoft.PowerShell.Core_00000000-0000-0000-0000-000000000000_en-US_HelpContent.zip', because Save-Help for  should download 'Microsoft.PowerShell.Core_00000000-0000-0000-0000-000000000000_en-US_HelpContent.zip', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 333
333:     $compressedFile | Should -Be $expectedCompressedFile -Because "Save-Help for $module should download '$expectedCompressedFile'"
```
Validate 'Save-Help -DestinationPath for all PowerShell modules..Validate Save-Help for the 'Microsoft.PowerShell.Host' module
```powershell
Expected 'Microsoft.PowerShell.Host_56d66100-99a0-4ffc-a12d-eee9a6718aef_en-US_helpcontent.zip', because Save-Help for  should download 'Microsoft.PowerShell.Host_56d66100-99a0-4ffc-a12d-eee9a6718aef_en-US_helpcontent.zip', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 333
333:     $compressedFile | Should -Be $expectedCompressedFile -Because "Save-Help for $module should download '$expectedCompressedFile'"
```
Validate 'Save-Help -DestinationPath for all PowerShell modules..Validate Save-Help for the 'Microsoft.PowerShell.Management' module
```powershell
Expected 'Microsoft.PowerShell.Management_eefcb906-b326-4e99-9f54-8b4bb6ef3c6d_en-US_helpcontent.zip', because Save-Help for  should download 'Microsoft.PowerShell.Management_eefcb906-b326-4e99-9f54-8b4bb6ef3c6d_en-US_helpcontent.zip', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 333
333:     $compressedFile | Should -Be $expectedCompressedFile -Because "Save-Help for $module should download '$expectedCompressedFile'"
```